crypto/s2n_certificate.c

```diff
@@ -678,8 +678,8 @@ int s2n_get_utf8_string_from_extension_data(const uint8_t *extension_data, uint3
     return S2N_SUCCESS;
 }
 
-static int s2n_parse_x509_extension(struct s2n_cert *cert, const uint8_t *oid_field_in, const uint32_t oid_field_in_len,
-                                      uint8_t *oid_value_out, uint32_t *oid_value_out_len, bool *critical)
+static int s2n_parse_x509_extension(struct s2n_cert *cert, const uint8_t *oid, const uint32_t oid_len,
```

Comment:
Is `oid_len` used by anything anymore? We're now using `OBJ_txt2obj` which takes in a null terminated c string.